src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -86,11 +85,6 @@ static DataTestUtility()
 
             System.Net.ServicePointManager.SecurityProtocol |= System.Net.SecurityProtocolType.Tls12;
 
-            if (TracingEnabled)
-            {
-                TraceListener = new TraceEventListener();
```

Comment:
We need this TraceListener to capture all the traces when `TracingEnabled` flag is on in config file.

Reply:
@karinazhou, I have two points to mention here:
1. Active trace listeners influence each other. You can get tests reproducible having only a single listener active at a time. As from now, we're getting a 2nd listener, we'll need to move this one somewhere closer to the test that uses it, and explicitly handle the listener's lifetime.
2. I explored the code but did not find anything that would use this listener, so I decided it is just a dangling code. Were my findings wrong?

Reply:
@winseros  This listener is to capture the traces for all the tests we are running. The TracingEnabled is by default turned off. This listener is for error analysis purposes. And now we are using reflection instead of a second listener for the Event Counter test so there will no impact on different active listeners.

Reply:
Regarding the recent changes and using reflection, this change is not directly related to the event counter, and it'd better be wrapped up in a separate PR for further discussions.
Feel free to create a new PR  and move it there with its related explanations, please.